src/pages/task/taskTabs/buildBaronAndAnnotations/AnnotationTicketsTable/AnnotationTicketRow.stories.tsx

```diff
@@ -2,7 +2,6 @@ import { CustomStoryObj, CustomMeta } from "test_utils/types";
 import { AnnotationTicketRow } from "./AnnotationTicketRow";
 
 export default {
-  title: "Pages/Task/BuildBaron/AnnotationTicketRow",
```

Comment:
I'm not super opposed to this because the nesting on Storybook gets kinda nasty... or we could reduce the nesting in the source code 😛

Reply:
My only gripe with this is that it makes it more difficult to trace stories to the actual file. I removed it from some of the other stories for consistency but we should reevaluate how we name our stories as a follow-up.